junit-mixin-plugin-test/src/test/scala/org/scalajs/junit/TraitToClassMixinTest.scala

```diff
@@ -0,0 +1,24 @@
+package org.scalajs.junit
+
+import org.junit._
+import org.junit.Assert._
+
+object TraitToClassMixinTest {
+  var count = 0
```

Comment:
Does JUnit guarantee that it doesn't execute tests in parallel within the same class? Also, does it guarantee that it will execute all the tests of a class in the same VM? Maybe `count` should be an `AtomicInteger`.

It seems really really fragile to have this global mutable state there. Also, it should probably be set to `0` in `beforeClass` if there's no other choice.